ui/lib/core/addon/templates/components/confirm/message.hbs

```diff
@@ -0,0 +1,41 @@
+
+  {{#maybe-in-element wormholeReference false}}
```

Comment:
Do we want to render the messages all of the time? Seems like we'd just want them when `showConfirm` is `true`. Though I guess they're only going to render after you open a dropdown, so it's not like there will be 100 of them at once so maybe it doesn't matter.

Reply:
i thought about this too, but rendering a `Message` all the time is what allows us to keep the 'Delete' button in the dropdown focused after clicking 'Cancel'. i've re-added in a conditional though so that the message contents (the overlay, icon, confirm & delete button) are only rendered when `showConfirm` is true, while the `Delete` button in the dropdown is always there (but disabled when the overlay is shown). let me know if you have a better suggestion!